docs/getting_started/tutorial.md

```diff
@@ -70,11 +78,11 @@ and a sample "search" app.
 Because the folder `mysite` was already created by `venv`, run `wagtail start` with an additional argument to specify the destination directory:
 
 ```console
-$ wagtail start mysite mysite
+$ wagtail start mysite
```

Comment:
```suggestion
$ wagtail start mysite mysite
```

@Dev-Yusuf this doesn’t seem correct? The line above explains why we give this `mysite` twice.